monkey/monkey_island/cc/services/attack/technique_reports/T1053.py

```diff
@@ -0,0 +1,35 @@
+from monkey_island.cc.services.attack.technique_reports import AttackTechnique
+from monkey_island.cc.database import mongo
+from common.utils.attack_utils import ScanStatus
+from common.data.post_breach_consts import POST_BREACH_JOB_SCHEDULING
```

Comment:
Ideally, all imports should follow the same structure:
```
future imports

python imports

third party imports

project imports
```
If not sure about why, look up PEP8.
You can automatically achieve this on pycharm with ctrl + alt + o.

Reply:
Maybe we should add https://pypi.org/project/flake8-import-order/